### PR TITLE
Create script

### PR DESCRIPTION
**Pull request by scott-williams-az**
### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-0000)
- [Unity reference site](https://asu.github.io/asu-unity-stack/)
- [Unity Design Kit](https://shared-assets.adobe.com/link/fb14b288-bf63-47e0-5d30-384de5560455)

### FOR APPROVERS

- [Percy build approval](https://percy.io/5eae92d9/-all-UDS-packages)

### CODE REVIEW

File: packages/unity-react-core/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "url": "https://github.com/ASU/asu-unity-stack/issues"
   },
   "scripts": {
+    "create": "plop",
```
Comment (davidornelas11) [inline packages/unity-react-core/package.json:31]: Should we name this to be a bit more descriptive of what the script does?

**Review by davidornelas11 — CHANGES_REQUESTED** [review]

File: packages/unity-react-core/package.json (hunk 1 above)
Comment (scott-williams-az) [reply packages/unity-react-core/package.json:31]: Right now it is only creating components. So we could rename to `create-component` however, we could expand this to create hooks or other patterns. So I am open to naming.

Comment (davidornelas11) [reply packages/unity-react-core/package.json:31]: Another option would be to add a flag to read arguments so we could do "--component" for components or "--hook"? I also don't want to extend this PR just to add functionality so we can add these notes to the ticket for further brainstorming and I can merge this

Comment (scott-williams-az) [reply packages/unity-react-core/package.json:31]: plop already takes care of this if we add more options

**Pull request merged**